src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -38,6 +43,12 @@ public MarkLettersPostedTask(LetterRepository repo,
         this.parser = parser;
     }
 
+
+    @Scheduled(cron = "${tasks.reports-cron}")
```

Comment:
perhaps best to be more specific and e.g. say it is `print-report-cron` or smth similar